client/components/Checkout.js

```diff
@@ -0,0 +1,457 @@
+                      onChange={handleChange}
+                    />
+                  </label>
+                  <label>
+                    Address:
+                    <input
+                      type="text"
+                      name="billingAddress"
+                      value={billingAddress}
+                      onChange={handleChange}
+                    />
+                  </label>
+                  <label>
+                    City:
+                    <input
+                      type="text"
+                      name="billingCity"
+                      value={billingCity}
+                      onChange={handleChange}
+                    />
+                  </label>
+                  <label>
+                    State:
+                    <input
+                      type="text"
+                      name="billingState"
+                      value={billingState}
+                      onChange={handleChange}
+                    />
+                  </label>
+                  <label>
+                    Zip Code:
+                    <input
+                      type="text"
+                      name="billingZip"
+                      value={billingZip}
+                      onChange={handleChange}
+                    />
+                  </label>
+                  <label>
+                    Email:
+                    <input type="text" name="email" value={email} onChange={handleChange} />
+                  </label>
+                  <label>
+                    Phone number:
+                    <input type="text" name="phone" value={phone} onChange={handleChange} />
+                  </label>
+                  {/* The below may be a placeholder for whatever is implemented for Square or Stripe */}
+                  <label>
+                    Credit Card Number:
+                    <input type="text" name="ccNumber" value={ccNumber} onChange={handleChange} />
+                  </label>
+                  <label>
+                    CCV:
+                    <input type="text" name="ccv" value={ccv} onChange={handleChange} />
+                  </label>
+                  <label>
+                    Credit Card Expiration Date:
+                    <input
+                      type="number"
+                      name="ccExpDateMonth"
+                      value={ccExpDateMonth}
+                      onChange={handleChange}
+                      min="1"
+                      max="12"
+                    />
+                    /
+                    <input
+                      type="number"
+                      name="ccExpDateYear"
+                      value={ccExpDateYear}
+                      onChange={handleChange}
+                    />
+                  </label>
+                </div>
+                <br />
+                <br />
+                <div>
+                  <h5>Shipping Information</h5>
+
+                  <label>
```

Comment:
My only suggestion here is that if the sameAddress box is checked, we don't even have to display the form below at all.